apps/ledger-live-mobile/src/screens/PairDevices/index.tsx

```diff
@@ -134,7 +134,7 @@ function PairDevicesInner({ navigation, route }: NavigationProps) {
       });
 
       try {
-        const transport = await TransportBLE.open(bleDevice);
+        const transport = await TransportBLE.open(bleDevice.id);
```

Comment:
this change on the (old)current pairing needs to be QAed to be safe ;)

Reply:
Not really, without the types it stupidly used the id skipping the string part, and not reusing the existing transport in cache. Now it should work better.